diff -urN tools/src/libbgl2txt.cpp.orig tools/src/libbgl2txt.cpp
--- tools/src/libbgl2txt.cpp.orig	2012-09-18 15:34:20.000000000 +0200
+++ tools/src/libbgl2txt.cpp	2012-09-18 15:34:25.000000000 +0200
@@ -2,6 +2,8 @@
 #include "bgl_babylonreader.h"
 #include "bgl_stardictbuilder.h"
 
+#include <cstring>
+
 void convert_bglfile(std::string infile, std::string source_charset, std::string target_charset)
 {
 	std::string outfile;

diff -urN tools/src/bgl_babylon.cpp.orig tools/src/bgl_babylon.cpp
--- tools/src/bgl_babylon.cpp.orig	2012-09-18 15:42:30.000000000 +0200
+++ tools/src/bgl_babylon.cpp	2012-09-18 15:42:59.000000000 +0200
@@ -25,6 +25,8 @@
 #include <glib/gstdio.h>
 #include <iconv.h>
 
+#include <cstring>
+
 #ifdef _WIN32
 #include <io.h>
 #define DUP _dup

diff -urN tools/src/bgl_stardictbuilder.cpp.orig tools/src/bgl_stardictbuilder.cpp
--- tools/src/bgl_stardictbuilder.cpp.orig	2012-09-18 15:45:30.000000000 +0200
+++ tools/src/bgl_stardictbuilder.cpp	2012-09-18 15:45:34.000000000 +0200
@@ -23,6 +23,8 @@
 #include <iostream>
 #include <glib.h>
 
+#include <cstring>
+
 StarDictBuilder::StarDictBuilder( std::string filename )
 {
   const char *p = strrchr(filename.c_str(), G_DIR_SEPARATOR);

diff -urN tools/src/dictbuilder.cpp.orig tools/src/dictbuilder.cpp
--- tools/src/dictbuilder.cpp.orig	2012-09-18 15:56:47.000000000 +0200
+++ tools/src/dictbuilder.cpp	2012-09-18 15:57:19.000000000 +0200
@@ -12,6 +12,8 @@
 #include <ctype.h>
 #include <arpa/inet.h>
 
+#include <cstring>
+
 #include "dictbuilder-tree.h"
 
 struct sectionEntry

diff -urN tools/src/tabfile2sql.cpp.orig tools/src/tabfile2sql.cpp
--- tools/src/tabfile2sql.cpp.orig	2012-09-18 16:00:31.000000000 +0200
+++ tools/src/tabfile2sql.cpp	2012-09-18 16:00:36.000000000 +0200
@@ -7,6 +7,8 @@
 
 #include <string>
 
+#include <cstring>
+
 void print_info(const char *info)
 {
 	g_print("%s", info);

diff -urN tools/src/Unihan.cpp.orig tools/src/Unihan.cpp
--- tools/src/Unihan.cpp.orig	2012-09-18 16:04:17.000000000 +0200
+++ tools/src/Unihan.cpp	2012-09-18 16:10:12.000000000 +0200
@@ -58,7 +58,7 @@
 	const char *han = line+2;
 
 	char *p;
-	p = strchr(han, '\t');
+	p = const_cast<char*>(strchr(han, '\t'));
 	if (!p) {
 		g_print("Error: %s\n", line);
 		return;
@@ -67,7 +67,7 @@
 	p++;
 	const char *key = p;
 
-	p = strchr(key, '\t');
+	p = const_cast<char*>(strchr(key, '\t'));
 	if (!p) {
 		g_print("Error: %s\n", line);
 		return;

diff -urN tools/src/KangXi.cpp.orig tools/src/KangXi.cpp
--- tools/src/KangXi.cpp.orig	2012-09-18 16:13:02.000000000 +0200
+++ tools/src/KangXi.cpp	2012-09-18 16:13:21.000000000 +0200
@@ -58,7 +58,7 @@
 	gint n = g_unichar_to_utf8(uc, utf8);
 	utf8[n] = '\0';
 	char *p;
-	p = strchr(line, '\t');
+	p = const_cast<char*>(strchr(line, '\t'));
 	if (!p) {
 		g_print("Error: %s\n", line);
 		return;

diff -urN tools/src/wubi.cpp.orig tools/src/wubi.cpp
--- tools/src/wubi.cpp.orig	2012-09-18 16:15:47.000000000 +0200
+++ tools/src/wubi.cpp	2012-09-18 16:15:50.000000000 +0200
@@ -6,6 +6,8 @@
 #include <glib.h>
 #include <string>
 
+#include <cstring>
+
 void convert(const char *filename)
 {
 	struct stat stats;

diff -urN tools/src/bgl2txt.cpp.orig tools/src/bgl2txt.cpp
--- tools/src/bgl2txt.cpp.orig	2012-09-18 16:26:24.000000000 +0200
+++ tools/src/bgl2txt.cpp	2012-09-18 16:26:38.000000000 +0200
@@ -1,5 +1,8 @@
 #include "libbgl2txt.h"
 
+#include <cstring>
+#include <cstdio>
+
 int main(int argc, char * argv[])
 {
 	if (argc<2) {

diff -urN tools/src/wordnet.cpp.orig tools/src/wordnet.cpp
--- tools/src/wordnet.cpp.orig	2012-09-18 16:28:12.000000000 +0200
+++ tools/src/wordnet.cpp	2012-09-18 16:28:27.000000000 +0200
@@ -3,6 +3,7 @@
 #include <glib.h>
 #include <string> 
 
+#include <cstring>
 
 static int hexalpha_to_int(int c)
 {
